airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/PartialConverter.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.read.cdc
+
+import io.debezium.spi.converter.ConvertedField
+import io.debezium.spi.converter.CustomConverter
+import io.debezium.spi.converter.RelationalColumn
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.util.concurrent.atomic.AtomicBoolean
+
+fun interface PartialConverter {
```

Comment:
what is a partial converter? could we have some documentation?

Reply:
this definitely needs comments, sorry about that

Reply:
done